app/models/certificate.server.ts

```diff
@@ -4,18 +4,19 @@ import { prisma } from '~/db.server';
 
 export type { Certificate } from '@prisma/client';
 
-export function getIssuedCertificateByUsername(username: Certificate['username']) {
-  /**
-   * There might be multiple certificates in the db for the same user, let's get
-   * the most recent one that has been successfully issued
-   */
+export async function getCertificateStatusByUsername(username: Certificate['username']) {
   return prisma.certificate
     .findMany({
-      where: { username, status: 'issued' },
-      orderBy: { validFrom: 'desc' },
+      where: {
+        username,
+        OR: [{ status: 'issued' }, { status: 'pending' }, { status: 'failed' }],
```

Comment:
This defines all 3 possible states, which implies that you don't actually need to include this.  Is there a reason you do?

Reply:
I was wondering how we handle certificate that has failed? I thought that we delete them if that happens, but I'm not quite sure...

Reply:
@dadolhay can confirm, but when we finish an order currently only have code to set `status=issued` - https://github.com/DevelopingSpace/starchart/blob/main/app/queues/certificate/order-completer-worker.server.ts#L76-L82.  It doesn't look to me like we have an error handler to set it equal to `failed` yet.  We were discussing this yesterday, and it hasn't been done, but it will be `status=failed`.

Reply:
What do you mean exactly? Don't you need an `OR` for conditional?

Reply:
If you're saying, `it's any of these` and those are the only values we allow, there's no point in doing the condition.  Every record in our DB is one of `issued`, `pending`, or `failed`, so this will return the whole set.

Reply:
oh wait you're right